overlord/devicestate/handlers_install.go

```diff
@@ -447,6 +492,22 @@ func (m *DeviceManager) doRestartSystemToRunMode(t *state.Task, _ *tomb.Tomb) er
 		return err
 	}
 
+	// write timing information, the lock must be unset for this
+	var chgIDs []string
+	for _, chg := range st.Changes() {
+		chgIDs = append(chgIDs, chg.ID())
```

Comment:
you should skip seed otherwise we will get it twice which is confusing, also still think this should go inside writeTimings with the lock handling as well

Reply:
maybe I should clarify this better:

snap debug timings --ensure=seed

will also print the corresponding change timings.  You can try locally with snap debug timings --ensure=auto-refresh

Reply:
Nice, thank you! I am fixing this now.